canvas/fizz-27-py/src/lib/courses.py

```diff
@@ -0,0 +1,33 @@
+from lib.canvasRequest import get
+import os
+from datetime import datetime
+
+def getCoursesByStatus(status):
+    admin_user_id = os.getenv("CANVAS_ADMIN_ID")
+    courses = get(f"/api/v1/accounts/{admin_user_id}/courses?state={status}")
+    return courses
+
+def getAvailableCourses():
+    return getCoursesByStatus("available")
+
+def getCompletedCourses():
+    return getCoursesByStatus("completed")
+
+def getCoursesForYear():
+    available_courses = getAvailableCourses()
+    completed_courses = getCompletedCourses()
+    total_courses = available_courses + completed_courses
+    return list(filter(filterCoursesByYear, total_courses))
```

Comment:
Suggestion to start using list comprehensions. Essentially the same as what you've done but considered "more pythonic".

```
return [tc for tc in total_courses if filterCoursesByYear(tc)]
```

If taking that approach, for readability I would change the function name to `course_in_report_year`.